src/main/java/de/retest/recheck/ui/descriptors/SutState.java

```diff
@@ -22,25 +25,42 @@
 	@XmlElement
 	private final List<RootElement> descriptors;
 
+	@XmlElement
+	private final Map<String, String> metaData;
```

Comment:
[Metadata](https://en.wikipedia.org/wiki/Metadata) is one word only:
```suggestion
	private final Map<String, String> metadata;
```

Reply:
Done